base/task.jl

```diff
@@ -3,7 +3,7 @@
 ## basic task functions and TLS
 
 # Container for a captured exception and its backtrace. Can be serialized.
-type CapturedException
+type CapturedException <: Exception
```

Comment:
@amitmurthy did you have a specific reason for making this not a subtype of Exception?